cpc/feature_loader.py

```diff
@@ -138,6 +142,9 @@ def getEncoder(args):
     elif args.encoder_type == 'lfb':
         from .model import LFBEnconder
         return LFBEnconder(args.hiddenEncoder)
+    elif args.encoder_type == 'smart':
```

Comment:
Same thing, remove this block. I made a mistake adding this.

Reply:
done